yklstartdeepdoc.py

```
@@ -0,0 +1,1095 @@
+            logger.info(f"  - 保存位置:      {images_dir}")
+        else:
+            logger.info(f"\n未检测到可提取的图表（layout_type 中无 figure/table 类型）")
+            logger.info(f"提示: 版面分析模型可能未检测到图片。尝试降低检测阈值或检查 PDF 质量。")
+
+        return saved, stats
+
+    def _merge_by_layoutno(self, bboxes: List[Dict]) -> List[Dict]:
+        """
+        将相同 layoutno 的文本块合并为完整段落。
+
+        RAGFlowPdfParser 的 _concat_downward (xgboost) 已重新启用，
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)